cmd/zpool/zpool_iter.c

```diff
@@ -504,12 +592,22 @@ all_pools_for_each_vdev_run(int argc, char **argv, char *cmd,
 void
 free_vdev_cmd_data_list(vdev_cmd_data_list_t *vcdl)
 {
-	int i;
+	int i, j;
 	for (i = 0; i < vcdl->count; i++) {
```

Comment:
nit: We're now able to switch the c99 style usage here for the variables.  It'd be nice to switch to this usage in functions touched by this patch.

```
for (int i = 0; i < vcdl->count; i++) {
```